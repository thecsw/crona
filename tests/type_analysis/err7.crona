foo : void(a : int, b : int) {}

main : void() {
    foo(true, false);
    foo("1", "2");
    foo(true, "hello");
}