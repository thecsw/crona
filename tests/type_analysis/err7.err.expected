FATAL [4,9]: Type of actual does not match type of formal
FATAL [4,15]: Type of actual does not match type of formal
FATAL [5,9]: Type of actual does not match type of formal
FATAL [5,14]: Type of actual does not match type of formal
FATAL [6,9]: Type of actual does not match type of formal
FATAL [6,15]: Type of actual does not match type of formal
Type Analysis Failed
